site/aktuelles/2019-05-trail-running-festival-innsbruck/index.html

```diff
@@ -1,24 +1,27 @@
 <div class="hero hero--image hero--small"
-style="background: linear-gradient(0deg,#FCC509A3,#FCC509B8),url('/assets/images/aktuelles/2019_05TrailFestivalInnsbruckSteffiSVM.jpg') top center / cover;">
+     style="background: linear-gradient(0deg,#FCC509A3,#FCC509B8),url('/assets/images/aktuelles/2019_05TrailFestivalInnsbruckSteffiSVM.jpg') top center / cover;">
```

Comment:
Hero image loogs not that good imo, but not that bad

Reply:
Agree. Noted for 2nd semester